modules/behavioural_qc/jsx/behaviouralQCIndex.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {TabPane, Tabs} from 'jsx/Tabs';
+import IncompleteForms from './tabs_content/incompleteForms';
+import DataConflicts from './tabs_content/dataConflicts';
+import BehaviouralFeedback from './tabs_content/behaviouralFeedback';
+
+/**
+ * Behavioural Quality Control.
+ *
+ * @description the Behavioural Quality Control of LORIS.
+ *
+ * @author Alizée Wickenheiser
+ * @version 1.0.0
+ *
+ */
+class BehaviouralQC extends Component {
+  /**
+   * Constructor of component
+   * @param {object} props - the component properties.
+   */
+  constructor(props) {
+    super(props);
+  }
+  /**
+   * @return {DOMRect}
+   */
+  render() {
```

Comment:
If this class only has a render function it can be a function type component and not a class one.